test/cmd/basicresources.sh

```diff
@@ -282,6 +282,18 @@ echo "OAuth descriptor: ok"
 
 os::cmd::expect_success 'oc delete all --all'
 
+os::test::junit::declare_suite_start "cmd/basicresources/get"
```

Comment:
I would probably put all of these (and the tests above) in a new script called `kubeconfig` or something